sdk/identity/identity/package.json

```diff
@@ -113,6 +113,7 @@
     "@azure/msal-browser": "^2.32.0",
     "@azure/msal-common": "^9.0.0",
     "@azure/msal-node": "^1.14.4",
+    "@azure/identity-common": "1.0.0-beta.1",
```

Comment:
can we remove some dependencies from identity now because of what has moved into common?